File: app/src/main/java/com/tt/handsomeman/ui/my_projects/MyProjectsFragment.java
package com.tt.handsomeman.ui.my_projects;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.tt.handsomeman.R;

public class MyProjectsFragment extends Fragment {
    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        View root = inflater.inflate(R.layout.fragment_my_projects, container, false);
        return root;
    }
}

File: app/src/main/java/com/tt/handsomeman/ui/my_projects/MyProjectsFragment.kt
package com.tt.handsomeman.ui.my_projects

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

import com.tt.handsomeman.R

class MyProjectsFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater,
                              container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_my_projects, container, false)
    }
}
